libctru/source/svc.s

```diff
@@ -259,6 +295,17 @@ svcGetProcessInfo:
 	ldr  r4, [sp], #4
 	bx   lr
 
+.global svcGetThreadInfo
+.type svcGetThreadInfo, %function
+svcGetThreadInfo:
```

Comment:
As I said on irc, I merely followed what was done for svcGetProcessInfo. I didn't test this one since we don't know what values we should get depending the the 3rd argument.
If it bothers you I'll comment it out